server/channels/app/post.go

```diff
@@ -492,6 +493,28 @@ func (a *App) FillInPostProps(c request.CTX, post *model.Post, channel *model.Ch
 		}
 	}
 
+	if len(ChannelMentionsAcrossTeams) > 0 {
+		for mentionedTeam, mentionedTeamChannels := range ChannelMentionsAcrossTeams {
+			team, err := a.Srv().Store().Team().GetByName(mentionedTeam)
+			if err != nil {
+				mlog.Warn("Failed to get team of the channel mention", mlog.String("mentionedTeam", channel.TeamId))
```

Comment:
Use `c.Logger().Warn(` instead of `mlog.Warn`. Let's do the change in L485 as well.